fs/accounting/token_bucket.go

```diff
@@ -126,7 +125,6 @@ func (tb *tokenBucket) StartTokenTicker(ctx context.Context) {
 	go func() {
 		for range ticker.C {
 			limitNow := ci.BwLimit.LimitAt(time.Now())
-			tb.currLimitMu.Lock()
 
 			if tb.currLimit.Bandwidth != limitNow.Bandwidth {
 				tb.mu.Lock()
```

Comment:
If we are using the tb.mu for protection of all the tb variables then we need to lock it before reading `tb.currLimit` so the tb.mu.Lock() needs to move to where the `tb.currLimitMu.Lock()` was. Same with the unlock.